languages/go/exercises/concept/basic-strings/.docs/introduction.md

```diff
@@ -0,0 +1,19 @@
+# Introduction
+
+### Strings
+
+A `string` in Go is a sequence of `bytes`, which doesn't necessarily have to represent characters.
+That being said, `UTF-8` is a central part of `strings` in Go. It is easy to convert a string to `runes` (`UTF-8` characters) or iterate over `runes` in a string.
+This makes dealing with different languages or other special characters in Go very simple.
+When dealing with `UTF-8` characters it is important to know, that not all characters have the same length.
+`ASCII` characters have the length of one `byte`. Other characters can have up to 4 bytes.
+Runes, bytes and their connection to strings will be handled more in-depth in a later exercise.
+
+Since `strings` are based on `[]byte` there are some commonalities. For example you can get a `byte` at position `i` from a `string` with `s[i]`. Other things are different than with a `slice`. For example can a string **not** be changed with `s[i] = 'a'`.
+
```

Comment:
Looks much better here, more clear definition of the topic. I would amend the language slightly as follows:

```
When dealing with `UTF-8` characters it is important to know that not 
all characters have the same length; `ASCII` characters have a length 
of one `byte`, while other characters can be as long as four `bytes`. 
Runes, bytes, and their connection to strings will be handled more in-depth 
in a later exercise.

The underlying type (aka representation) for `string` in go is the `[]byte` slice, 
which allows for some commonalities with slices in general. For example, you 
can get a `byte` at position `i` from a `string` called `s` with sub-indexing, like 
`s[i]`. With that said, `string` types are immutable so operations like `s[i] = 'a'` 
will not have any impact to the underlying slice.

...
```

Reply:
Very nice! Will use this.

One little comment on the last sentence: Maybe it could be more straight forward by mentioning that it is not allowed / not possible / won't compile.

Maybe: 
> With that said, `string` types are immutable so operations like `s[i] = 'a'` are not available and will result in a compilation error.